mirdata/datasets/acousticbrainz_genre.py

```diff
@@ -397,12 +371,23 @@ class Dataset(core.Dataset):
     The acousticbrainz genre dataset
     """
 
-    def __init__(self, data_home=None, index=None):
+    def __init__(
+        self,
+        data_home=None,
+        remote_index=None,
+        remote_index_name=None,
+    ):
+        if remote_index and remote_index_name:
+            data = core.LargeData(remote_index_name, remote_index=remote_index)
+            index = data.index
+        else:
+            index = None
```

Comment:
Not covered by tests apparently

Reply:
many of the lines of acousticbrainz_genre need to be checked because of the remote index. I opened #437 to address it